cube_helper/fix_known.py

```diff
@@ -0,0 +1,180 @@
+# (C) Crown Copyright, Met Office. All rights reserved.
+#
+# This file is part of cube_helper and is released under the
+# BSD 3-Clause license.
+# See LICENSE in the root of the repository for full licensing details.
+from abc import ABCMeta, abstractmethod
+import six
+
+import iris
+
+from cube_helper.logger import log_module
+
+
+@six.add_metaclass(ABCMeta)
+class FixKnownIssue:
+    """
+    An abstract base class to identify and fix known issues with data loaded
+    into an `iris.cube.Cube`.
+
+    Args:
+        cube: the cube to check, and if necessary fix.
+    """
+    def __init__(self, cube):
+        self.cube = cube
+        self.logger = log_module()
+
+    @abstractmethod
+    def is_fix_needed(self):
+        """
+        Check if this fix applies to the object's cube.
+
+        Returns:
+            True if this fix should be applied to the cube that it was
+            instantiated with.
+        """
+        pass
+
+    @abstractmethod
+    def fix_cube(self):
+        """
+        Apply the fix to the object's cube and warn about the name of the fix
```

Comment:
Is `warn` the correct word to use here? If so, `info` on L45 below should be `warning`.

Reply:
Changed warn to inform in d049090. Further clarified in the docstring the contents of the message in 8f85421.